File: app/src/main/java/com/ddiehl/android/htn/listings/comments/LinkCommentsView.java
package com.ddiehl.android.htn.listings.comments;

import com.ddiehl.android.htn.listings.ListingsView;
import com.ddiehl.android.htn.listings.links.LinkView;

public interface LinkCommentsView extends ListingsView, LinkView, CommentView {

    String getCommentId();

    String getArticleId();

    String getSubreddit();

    String getSort();

    void refreshOptionsMenu();
}

File: app/src/main/java/com/ddiehl/android/htn/listings/comments/LinkCommentsView.kt
package com.ddiehl.android.htn.listings.comments

import com.ddiehl.android.htn.listings.ListingsView

interface LinkCommentsView : ListingsView {

    val subreddit: String

    val articleId: String

    val commentId: String?

    var sort: String

    fun refreshOptionsMenu()
}
